sw/device/silicon_creator/manuf/base/sram_ft_individualize.c

```diff
@@ -267,9 +215,14 @@ static status_t patch_ast_config_value(void) {
  * all fields can be programmed until the personalization stage.
  */
 static status_t provision(ujson_t *uj) {
+  // Enable ATE GPIO indicators.
+  TRY(configure_ate_gpio_indicators());
+
   // Get host data.
   LOG_INFO("Waiting for FT SRAM provisioning data ...");
```

Comment:
@timothytrippel I think the LOG_INFO will be problematic on simulation.

Reply:
These LOG_INFO statements are unused by sim environments, and when this gets compiled for a sim environment these should be skipped: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/runtime/log.h;l=135?q=LOG_INFO&ss=opentitan%2Fopentitan.

They would only be used by host SW that polled the console for log statements.

Is that what you are concerned about @sha-ron ? or are you concerned about some other aspect of these?

Reply:
Yes, this was my concern. Is it true also for test_status_set that will be skipped when compiled for a sim environment?